[update] Update cordova blank screen.

vikasjayaram opened the pull request:
### Your checklist for this pull request

Please review the [guidelines for contributing](https://github.com/auth0/docs#contributing) to this repository.
- [x] Commit messages conforms to our [commit message standards](https://github.com/auth0/docs#commit-messages)
- [x] Content conforms to our [Contributing Guidelines](https://github.com/auth0/docs#contributing-guidelines)
- [x] If applicable, you have added details to the [update feed](https://github.com/auth0/docs/tree/master/updates) 
### Description

Please describe your pull request.

💔Thank you!
Added trouble shooting note that setting sso to true or missing configuration for lock can cause blank screen pop up in cordova.

vikasjayaram committed:
Update cordova blank screen.

Added trouble shooting note that setting sso to true or missing configuration for lock can cause blank screen pop up in cordova.

ntotten commented on articles/native-platforms/cordova.md:
````diff
@@ -119,3 +119,14 @@ cp plugins/cordova-plugin-inappbrowser/src/android/* platforms/android/[yourAppN
+```
+      lock.show(function(err, profile, token) {
+      });
+```
````
There shouldn't be indentation inside the code block. It should be 

```
lock.show(function(err, profile, token) {
});
```

ntotten commented on articles/native-platforms/cordova.md:
````diff
@@ -119,3 +119,14 @@ cp plugins/cordova-plugin-inappbrowser/src/android/* platforms/android/[yourAppN
+You must configure the lock as shown below, to avoid the blank screen popup with the OK button.
+```
+      lock.show({sso: false},function(err, profile, token) {
+      });
````
same

ntotten commented on articles/native-platforms/cordova.md:
```diff
@@ -119,3 +119,14 @@ cp plugins/cordova-plugin-inappbrowser/src/android/* platforms/android/[yourAppN
 #### Get a blank page with an OK after signin
 
 This means that the `InAppBrowser` plugin wasn't installed successfully by Cordova. See the previous section to learn how to solve this.
```
Should the "See the previous section to learn how to solve this." still be here since you have shown below how to solve?

vikasjayaram committed:
Update Blank Page Section Content

Removed the following line
"This means that the InAppBrowser plugin wasn't installed successfully by Cordova. See the previous section to learn how to solve this."

Removed indentation from code sample.
